python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -561,7 +561,11 @@ def run(self, sensorRef, templateIdList=None):
                 if not self.config.doDipoleFitting:
                     self.measurement.run(diaSources, subtractedExposure)
                 else:
-                    self.measurement.run(diaSources, subtractedExposure, exposure, templateExposure)
+                    if self.config.doSubtract:
```

Comment:
If `doSubract == True` then we have a matchedExposure and we can use that rather than computing it from `exposure - subtractedExposure`. This is noted in the `DipoleFitTask.run()` docstring.